develop/dev-guide-proxysql-integration.md

```diff
@@ -1,682 +1,1115 @@
+
+1. Start Docker. If Docker has already started, skip this step:
+
+    <SimpleTab groupId="os">
+
+    <div label="macOS" value="macOS">
+
+    Double-click the icon of the installed Docker to start it.
+
+    </div>
+
+    <div label="CentOS" value="CentOS">
+
+    ```bash
+    systemctl start docker
+    ```
+
+    </div>
+
+    <div label="Windows" value="Windows">
+
+    Double-click the icon of the installed Docker to start it.
+
+    </div>
+
+    </SimpleTab>
+
+2. Pull down the ProxySQL Docker image and start a ProxySQL container:
+
+    <SimpleTab groupId="os">
+
+    <div label="macOS" value="macOS">
+
+    ```bash
+    docker compose up -d
+    ```
+
+    </div>
+
+    <div label="CentOS" value="CentOS">
+
+    ```bash
+    docker compose up -d
+    ```
+
+    </div>
+
+    <div label="Windows (Git Bash)" value="Windows">
+
+    ```bash
+    docker compose up -d
+    ```
+
+    </div>
+
+    </SimpleTab>
+
+3. Integrate with ProxySQL by running the following command, which will execute `proxysql-prepare.sql` inside **ProxySQL Admin Interface**:
+
+    <SimpleTab groupId="os">
+
+    <div label="macOS" value="macOS">
+
+    ```bash
+    docker compose exec proxysql sh -c "mysql -uadmin -padmin -h127.0.0.1 -P6032 < ./proxysql-prepare.sql"
+    ```
+
+    </div>
+
+    <div label="CentOS" value="CentOS">
+
+    ```bash
+    docker compose exec proxysql sh -c "mysql -uadmin -padmin -h127.0.0.1 -P6032 < ./proxysql-prepare.sql"
+    ```
+
+    </div>
+
+    <div label="Windows (Git Bash)" value="Windows">
+
+    ```bash
+    docker compose exec proxysql sh -c "mysql -uadmin -padmin -h127.0.0.1 -P6032 < ./proxysql-prepare.sql"
+    ```
+
+    </div>
+
+    </SimpleTab>
+
     > **Note:**
     >
-    > - The preceding command uses `unistore` as the storage engine, which is a test storage engine in TiDB. Make sure that you use it in a test environment only.
-    > - `TIDB_SERVER_PATH`: the path of the compiled binary using `go build`. For example, if you execute the previous command under `/usr/local`, `TIDB_SERVER_PATH` is `/usr/local/tidb/tidb-server/tidb-server`.
-    > - `LOCAL_TIDB_LOG`: the log file path of TiDB.
+    > The proxysql-prepare.sql script does the following:
+    >
+    > 1. Adds a user using the username and password of your cluster.
+    > 2. Assigns the user to the monitoring account.
+    > 3. Adds your TiDB Serverless Tier cluster to the list of hosts.
+    > 4. Enables a secure connection between ProxySQL and TiDB Serverless.
+    >
+    > To have a better understanding, it is strongly recommended that you review the `proxysql-prepare.sql` file. To learn more about ProxySQL configuration, see [ProxySQL documentation](https://proxysql.com/documentation/proxysql-configuration/).
 
-</div>
+    The following is an example output. You will see that the hostname of your cluster is shown in the output, which means that the connectivity between ProxySQL and the TiDB Serverless Tier cluster is established.
```

Comment:
```suggestion
    The following is an example output. You will see that the hostname of your cluster is shown in the output, which means that the connection between ProxySQL and the TiDB Serverless Tier cluster is established.
```

Reply:
just the connectivity is established, the connection will be established in "Step 4. Connect to your TiDB cluster through ProxySQL"